SQL/New_patches/2018-05-18_adding_physiological_all_sites_permissions.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO permissions 
+    (
+        code, 
+        description, 
+        categoryID
+    ) VALUES (
+        'electrophysiology_browser_view_allsites',
+        'View all-sites Electrophysiology Browser pages',
+        (SELECT ID FROM permissions_category WHERE Description='Permission')
+    );
+
+INSERT INTO permissions
+    (
+        code,
+        description,
+        categoryID
+    ) VALUES (
+        'electrophysiology_browser_view_site',
+        'View own site Electrophysiology Browser pages',
+        (SELECT ID FROM permissions_category WHERE Description='Permission')
+    );
+
+INSERT INTO user_perm_rel
+    (
+      UserID,
+      PermID
+    ) VALUES (
+      (SELECT ID FROM users WHERE UserID='admin'),
+      (SELECT permID FROM permissions WHERE code='electrophysiology_browser_view_allsites')
+    );
```

Comment:
Not going to block over it because the performance impact is so minimal for a one-off patch, but I think it would be better  to write this as an `INSERT INTO (UserID, PermID) SELECT xxx` statement with a join.